lib/Mojo/Server/Daemon.pm

```diff
@@ -179,9 +190,12 @@ sub _listen {
   );
 
   return if $self->silent;
+  my $aport
+    = $self->ioloop->acceptor($self->acceptors->[$#{$self->acceptors}])->port;
+  $url->port($aport) if !$url->port;
+  $url->host("*") if !$url->host;
   $self->app->log->info(qq{Listening at "$url"});
   $query->pairs([]);
-  $url->host('127.0.0.1') if $url->host eq '*';
```

Comment:
Why do you want beginners to see `Server available at http://*:3000` instead of `Server available at http://127.0.0.1:3000` when they start their morbo development server for the first time?